index.html

```diff
@@ -447,6 +447,13 @@ <h3>The <dfn>PerformanceResourceTiming</dfn> Interface</h3>
 method</a> [[!BEACON]];</li>
 <li><dfn>"other"</dfn>, if none of the above conditions match.</li>
 </ul>
+<aside data-dfn-for="PerformanceResourceTiming" class="note">
+The "navigation" initiator type includes all types of
+<a data-cite="!FETCH/#navigation-request">navigation requests</a>, including
+<a data-cite=
+"!service-workers-1/#ref-for-service-worker-registration-navigation-preload-enabled-flag%E2%91%A2">
```

Comment:
This reference doesn't resolve in SW spec. 

On a related note, should Fetch have this note? Seems like we're correcting for lack of clarify in Fetch spec with respect to navigation preload?

Reply:
Interesting.. Nope. Also, if I search for "preload" there are no hits in the doc. Hmm?